src/counts.jl

```diff
@@ -237,20 +237,25 @@ Add counts based on `x` to a count map. New entries will be added if new values
 If a weighting vector `wv` is specified, the sum of the weights is used rather than the
 raw counts.
 
-alg can be one of
-    :auto       -   default; uses :radixsort if safe to do so
-    :radixsort  -   if radixsort_safe(eltype(x)) == true then use radixsort to
-                    sort the input array in order to achieve faster performance
-                    by using more RAM. One should choose :dict if the
-                    amount of available RAM is a limitation.
-    :dict       -   Uses Dict-based method which is generally slower but uses
-                    less RAM and is safe for any data type.
+`alg` can be one of
+- `:auto`:      default; uses `:radixsort` if safe to do so
+
+- `:radixsort`: if `radixsort_safe(eltype(x)) == true` then use `:radixsort`
```

Comment:
"use `:radixsort`" is kind of a circular explanation. Should be "radix sort", ideally with a link to e.g. Wikipedia. It would also make sense to be more specific than "by using more RAM": IIRC this requires making a copy of the input vector.